### ISSUE TITLE
SVGs don't match colour of parent text

### ISSUE THREAD

**Issue by gregtyler** [ministryofjustice/moj-frontend#162]
Where we use inline SVGs (i.e. [Primary navigation with toggle search](https://moj-design-system.herokuapp.com/components/primary-navigation#toggle-search), [Rich text editor](https://moj-design-system.herokuapp.com/components/rich-text-editor) and [Timeline](https://moj-design-system.herokuapp.com/components/timeline)), the colour of the SVG icons is hard-coded into the SVG files themselves.

This means they don't adapt to the text colour around them changes, which makes the icons disappear in certain visual modes. For example, in Windows High Contrast mode, the background is changed to black and any black icons cannot be seen.

Instead, the SVGs should have the CSS attribute `fill: currentColor` on them so that they match the surrounding text colour. The individual SVGs will also need to be updated to remove the hard-coded fill colour.
